docs/introduction/faq.md

```diff
@@ -0,0 +1,216 @@
+
+### Lotus or Go-Filecoin, which is better for miners?
+
+Lotus is our reference implementation, and getting more developer attention right now.
+
+### What is your recommendation on the right hardware to use?
+
+While the Filecoin team does not recommend any specific hardware configuration, [we did share some setups](https://filecoin.io/blog/announcing-testnet-incentives/#hardware) we’ve used for various types of testing. However, it is overwhelmingly likely that there are more efficient setups, and we strongly encourage miners to test and experiment to find the best combinations.
+
+### We are worried about the ability for our network to handle the additional overhead of running a Filecoin node and still provide fast services for our customers. What are the computational demands of a Lotus node? Are there any metrics for node performance given various requirements?
+
+As we approach launch, our Testground team is prioritizing getting metrics to understand the performance of Lotus nodes; we will share these as they are determined. We are also still tweaking node and network design parameters, so these numbers will continue to improve - they will likely be orders of magnitude better by launch.
+
+### We bought a lot of hard drives of data through the Discovery project. When will they be shipped to China?
+
+There are a number of details that are still being finalized between the verified deals construction and the associated cryptoeconomic parameters.
+
+Our aim is to allow these details to finalize before shipping, but given timelines, we’re considering enabling teams to take receipt of these drives before the parameters are set.
+
+### Is there a miner’s KYC certification process, if so how does that work? How will you verify the mining location of miners?
+
+To receive tokens, miners will be required to verify their identity by providing passports or similar identity documents. We may need to request additional information in rare cases.
+
+We’ll use a variety of technical and non-technical mechanisms. They may also be different for different miners. To avoid cheating, we’re not revealing the specific mechanisms until verification begins.
+
+When we verify miners’ locations, we’ll reach out by email and coordinate a time that the miner will be standing by to provide verification. We’ll ask for some specific evidence and require submission within a short time period.
+
+Location verification is not required to participate in either (i) the global leaderboard or (ii) the regional leaderboard for the most competitive region (i.e., the region with the most storage).
+
+We consider the location of the storage and sealing hardware to be the location of the miner.
+
+Please note that miners who attempt to “spoof” their location will be completely ineligible for all rewards.
+
+### How will Filecoin identify the mining location of miners?
```

Comment:
Delete this question and succeeding paragraphs as it is a duplicate of the former question/answer